modules/using-cli.adoc

```diff
@@ -0,0 +1,221 @@
+
+* Use the WebSocket Secure (`wss`) protocol.
+To use `wss`, prefix the address with *`wss://`*, and
+* Add the port number after the address, for example,
++
+[source,terminal]
+----
+$ roxctl sensor generate k8s --central wss://stackrox-central.example.com:443
+----
+====
+
+[discrete]
+=== Downloading sensor bundle for existing clusters
+
+Use the following command to download sensor bundles for existing clusters by specifying a cluster name or ID.
+
+[source,terminal]
+----
+$ roxctl sensor get-bundle <cluster-name-or-id>
+----
+
+[discrete]
+=== Deleting cluster integration
+
+[source,terminal]
+----
+$ $ roxctl -e "$ROX_CENTRAL_ADDRESS" cluster delete --name=<cluster-name>
+----
+
+[IMPORTANT]
+====
+Deleting cluster integration will not remove {product-title} services running in the cluster.
+You can remove them by running the `delete-sensor.sh` script from the sensor installation bundle.
+====
+
+[id="check-policy-compliance_{context}"]
+== Checking policy compliance
+
+[discrete]
+=== Checking deployment YAML files
+
+The following command checks build-time and deploy-time violations of your security policies in YAML deployment files.
+//TODO: Add link to security policies section
+Use this command to validate:
+
+* Configuration options in a YAML file, such as resource limits or privilege options;
+or
+* Aspects of the images used in a YAML file, such as components or vulnerabilities.
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" deployment check --file=<yaml-filename>
+----
+
+[discrete]
+=== Checking images
+
+The following command checks build-time violations of your security policies in images.
+//TODO: Add link to security policy section
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" image check --image=<image-name>
+----
+
+[discrete]
+=== Checking image scan results
+
+You can also check the scan results for specific images.
+
+The following command returns the components and vulnerabilities found in the image in JSON format.
+The format is defined in the API reference.
+//TODO: Add link to the API reference.
+
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" image scan --image <image-name>
+----
+
+To cause {product-title} to re-pull image metadata and image scan results from the associated registry and scanner, add the `--force` flag.
+
+[NOTE]
+====
+To check specific image scan results, you need a token with both `read` and `write` permissions for the `Image` resource.
+The default *Continuous Integration* system role already has the required permissions.
+//TODO: Add link to the system role topic.
+====
+
+[id="debug-issues_{context}"]
+== Debugging issues
+
+[discrete]
+=== Managing Central log level
+
+Central saves information to its container logs.
+
+[discrete]
+==== Viewing the logs
+You can see these logs by running:
+
+.Kubernetes
+[source,terminal]
+----
+$ kubectl logs -n stackrox <central-pod>
```

Comment:
This might go back to my question on what's all supported with ACS, but normally we just show OpenShift/oc commands and not necessarily kubectl commands. Just wondering if this can be removed, or if it needs to be kept.

Same question on another section that also lists a Kubernetes option.